resources/views/rent.blade.php

```diff
@@ -0,0 +1,148 @@
+
+@extends("layouts.master")
+
+@section('body')
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <script type="text/javascript">
+        // Globals
+        var curLocation = 0;
+        var shapes = [$.parseJSON(<?php echo json_encode($shapes); ?>)];
+        var locations = <?php echo json_encode($locations); ?>;    
+
```

Comment:
Also, just FYI but the double curly braces work for echoing stuff out: `{{ son_encode($shapes) }}`